packages/components/message-box/src/index.vue

```diff
@@ -316,6 +329,9 @@ export default defineComponent({
       if (props.closeOnHashChange) {
         on(window, 'hashchange', doClose)
       }
+      if (props.draggable) {
+        useDraggable(rootRef, headerRef)
+      }
```

Comment:
**Suggestion:** I think we should make this reactive, when the value was set to false, we should be able to turn this feature off.

Reply:
You can enable watching in this `onMounted` block, also I didn't see your destroy handler, would you mind to add that?